common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -296,7 +296,7 @@ trait CommercialSwitches {
   val highMerchandisingComponentSwitch = Switch(
     SwitchGroup.Commercial,
     "render-commercial-high-slot-always",
-    "If on, the commercial-high slot will always render on pageload",
+    "If on, wil be a server side check of page keywords with high-merchandising keywords, before high-merch slot render.",
```

Comment:
There is a little typo there, but it might be slightly clearer to reword that as "If on, server will check tags for high-merchandising target before rendering high-merch slot."